packages/material/scss/popup/_theme.scss

```diff
@@ -47,8 +47,10 @@ $grouping-border: $grouping-header-bg !default;
         }
     }
 
-    // Tooltip popup container padding
-    .k-animation-container-shown.k-tooltip-wrapper {
+    // Tooltip and Notification popup container padding
+    .k-animation-container-shown.k-tooltip-wrapper,
+    .k-animation-container-shown {
+        .k-notification &,
```

Comment:
Seems like this targets the `.k-notification .k-animation-container-shown` class, which seems to be missing in the markup. Perhaps the rule is not needed, after all?